### PR TITLE
Authentication Pages

### PR DESCRIPTION
**Pull request by aniketxparihar**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -3,13 +3,22 @@ import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
 import { makeServer } from "./server";
-
+import { BrowserRouter } from "react-router-dom";
+import Router from "./Router"
+import {ThemeProvider} from "../src/Context/Theme-Context";
```
Comment (mohammadkhizerkhan) [inline src/index.js:8]: use barrel exporrt

**Review by mohammadkhizerkhan — APPROVED** [review]
Good folder Structure
Good naming convention used

File: src/Pages/Auth/Login/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
+import React, { useReducer, useEffect } from "react";
+import { Link } from "react-router-dom";
+import { useAuth } from "../../../Context/Auth-Context";
```
Comment (shraddha-amber) [inline src/Pages/Auth/Login/Login.jsx:3]: could do barrel export for all the contexts

File: src/Pages/Auth/Signup/Signup.jsx [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+            >
+              Password
+            </label>
+            <input
+              type="password"
+              id="password__input"
+              className="password__input txt-2xl"
+              value={state.password}
+              onChange={(e) =>
+                dispatch({ type: "PASSWORD", payload: e.target.value })
```
Comment (shraddha-amber) [inline src/Pages/Auth/Signup/Signup.jsx:118]: always pass payload value inside object

**Review by shraddha-amber — APPROVED** [review]
The code structure is good
Always provide some description in pr about the changes you made

**Pull request merged**